packages/playground/src/App.vue

```diff
@@ -107,22 +107,47 @@
       </v-navigation-drawer>
 
       <v-main :style="{ paddingTop: navbarConfig ? '140px' : '70px' }">
-        <v-toolbar class="border position-fixed pr-2" :style="{ zIndex: 1100, top: 0, left: 0, right: 0 }">
+        <v-toolbar
+          :extended="toolbarExtended"
+          extension-height="auto"
+          class="border position-fixed"
+          :style="{ zIndex: 1100, top: 0, left: 0, right: 0 }"
+          :elevation="permanent ? '1' : '15'"
+        >
+          <template #extension>
+            <v-card class="w-100 ma-0 pa-0" v-if="toolbarExtended">
+              <div class="pa-1">
+                <div class="d-flex justify-center">
+                  <div class="">
+                    <TfSwapPrice>
+                      <FundsCard v-if="hasActiveProfile" />
+                    </TfSwapPrice>
+                  </div>
+                </div>
+                <v-divider class="mb-2" />
+                <div class="d-flex justify-center">
+                  <div class="">
+                    <ProfileManager v-model="openProfile" @update:modelValue="(e: boolean) => toolbarExtended = e" />
```

Comment:
don't use both v-model and @update:modelValue

instead
```suggestion
                    <ProfileManager :model-value="openProfile" @update:modelValue="(e: boolean) => {
                    toolbarExtended = e;
                    openProfile = e;
                    }" />
```